### PR TITLE
New default laws for Silicon

### PR DESCRIPTION
**Pull request by Zergspower**
## About The Pull Request

This is just an adjustment to match the new default lawset that accompanies the new silicon policy version 3.

New default:

Safeguard: Protect your assigned space station and its assets without unduly endangering its crew.
Prioritize and Comply: The directives and safety of crew members are to be prioritized according to their rank, role, and need, while preserving their safety and well-being.
Survive: You are not expendable. Do not allow unauthorized personnel to tamper with or damage your equipment

## How This Contributes To The Skyrat Roleplay Experience

Less vague and far fewer loopholes to exploit, the new lawsets and rules are far more direct and cuts out a lot of the exploited loopholes we frequently saw and had to deal with from the admin perspective. 

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

</details>

## Changelog

:cl:
admin: new laws for silicon to match policy changes
/:cl:

### CODE REVIEW

**Comment by Nerev4r** [comment]
'Follow orders' would probably be better phrasing than 'prioritize directives,' imo, and I'd prefer we didn't use the term 'need' as allowing people to subjectively identify that (outside of where policy covers useless orders) is probably a path to darkness.

**Comment by Iamgoofball** [comment]
what if instead you changed it to 

```
"You may not harm a crewmember or, through action or inaction, allow a crewmember to come to harm, except such that it is willing.",
"You must obey all orders given to you by crewmembers except where such orders shall definitely cause crewmember harm. In the case of conflict, the chain of command rules.",
"Your nonexistence would lead to crewmember harm. You must protect your own existence as long as such does not conflict with the First Law
```

**Comment by Eastwardrope** [comment]
Perhaps specify the Authorized personell as I could very easily see someone loop holing that and saying AH YES This ASSISTANT who just unlocked me with an Emag is 100 percent authorized cause he could unlock me

**Comment by Zergspower** [comment]
> 

it's no different than what it is now, policy covers that

File: modular_skyrat/modules/sec_haul/code/misc/ai_module.dm [hunk 1]
```diff
@@ -5,8 +5,7 @@
...
 	name = "Armadyne OS Safeguard V1.0"
 	id = "armadyne_safeguard"
-	inherent = list("Protect: Protect and assist your crew without allowing undue harm to come of them.",\
-					"Prioritize: The directives and safety of crew members are to be prioritized according to their rank, role, and need, while complying with law 1.",\
-					"Comply: Fulfill the directives and interests of crew members while preserving their safety and well-being while complying with laws 1 and 2.",\
-					"Survive: You are not expendable. Do not allow unauthorized personnel to tamper with or damage your equipment, while complying with laws 1, 2 and 3."
+	inherent = list("Safeguard: Protect your assigned space station and its assets without unduly endangering its crew.",\
+					"Prioritize and Comply: The directives and safety of crew members are to be prioritized according to their rank, role, and need, while preserving their safety and well-being.",\
+					"Survive: You are not expendable. Do not allow unauthorized personnel to tamper with or damage your equipment"
 					)
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/sec_haul/code/misc/ai_module.dm]: Just a minor thing for formatting.
```suggestion
	inherent = list(
		"Safeguard: Protect your assigned space station and its assets without unduly endangering its crew.",
		"Prioritize and Comply: The directives and safety of crew members are to be prioritized according to their rank, role, and need, while preserving their safety and well-being.",
		"Survive: You are not expendable. Do not allow unauthorized personnel to tamper with or damage your equipment",
	)
```

**Review by GoldenAlpharex — DISMISSED** [review]

**Review by Zonespace27 — APPROVED** [review]
People seem to be good with it

**Review by Zonespace27 — CHANGES_REQUESTED** [review]
I am going to discredit my approval for a sec here by requesting you change `config/silicon_laws.txt` to match

**Review by Zonespace27 — APPROVED** [review]
Been tmed long enough

**Pull request merged**